python/tk_syntheyes/inbuilt_app.py

```diff
@@ -0,0 +1,27 @@
+from engine import SynthEyesEngine
+
+class InbuiltApp(object):
+    @property
+    def display_name(self):
+        return ""
+    
+    @property
+    def description(self):
+        return ""
+        
+    @property
+    def commands(self):
+        return {
+            # "app_name":
```

Comment:
Den auskommentierten Code gern raus

Reply:
Der auskommentierte Code hier ist als Hilfe für die potentielle zukünftige Implementierung weitere Features gedacht gewesen. Ich hab' den Kommentar jetzt entfernt und wie bei einigen anderen Dateien von SGTK auch, das Beispiel in die Funktionsbeschreibung gepackt. So macht's auch mehr Sinn.